components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -829,7 +829,7 @@ export default class SuggestionBox extends React.PureComponent {
                     onSelect={this.handleSelect}
                 />
                 {(this.props.openWhenEmpty || this.props.value.length >= this.props.requiredCharacters) && this.state.presentationType === 'text' &&
-                    <div style={{width: this.state.width}}>
+                    <div>
```

Comment:
since the inline style got removed I think we can also get rid of the wrapping div